lib/packages/crypto/src/getAccountIdFromPublicKey.ts

```diff
@@ -12,11 +11,35 @@ import {Big} from 'big.js';
 export const getAccountIdFromPublicKey = (publicKey: string): string => {
     const hash = CryptoJS.SHA256(CryptoJS.enc.Hex.parse(publicKey));
     const bytes = Converter.convertWordArrayToByteArray(hash);
-    const slice = bytes.slice(0, 8);
-
-    const result = slice.reduce(
-        (acc, num, index) => acc.add(Big(num).mul(Big(2).pow(index * 8))),
-        Big(0)
-    );
-    return result.toFixed();
+    let result = '';
+    for (let i = 0; i < 8; i++) {
+        let byte = bytes[i].toString(16);
+        if (byte.length < 2) { byte = '0' + byte; }
+        result = byte + result;
+    }
+    return hexToDec(result);
 };
+
+/**
+ * Arbitrary length hexadecimal to decimal conversion
+ * https://stackoverflow.com/questions/21667377/javascript-hexadecimal-string-to-decimal-string
+ * @param s A hexadecimal string
+ * @return A decimal string
+ */
+function hexToDec(s) {
```

Comment:
If we could put this in another file or add to Converter we could reuse this, when needed. Furthermore,  this method needs unit tests. Coverage is lowering with this PR